mainnet/README.md

```diff
@@ -190,6 +190,10 @@ The following command can be used to opt out from the EigenDA AVS:
 
 ## Upgrade your node
 
+Please monitor the following channels for updates to EigenDA Operator software:
+- [EigenLayer Discord](https://discord.gg/eigenlayer): #support-operators channel.
```

Comment:
Are we not sharing the operator working group Slack channel as well https://eigenlabs.slack.com/archives/C05SQ99UQ64 ?